src/components/shared/upload/Upload.tsx

```diff
@@ -173,17 +171,16 @@ export default function Upload(props: UploadProps) {
     function finishUpUpload(finalFinalName: string, importOptionsPrep: string) {
         let keyToSend = key;
         if (!keyToSend) keyToSend = null;
-        getUploadCredentialsAndId(UploadHelper.getProjectId(), finalFinalName, uploadFileType, importOptionsPrep, UploadType.DEFAULT, keyToSend, (results) => {
-            const credentialsAndUploadId = JSON.parse(JSON.parse(results.data['uploadCredentialsAndId']));
+        getUploadCredentialsAndId(UploadHelper.getProjectId(), finalFinalName, uploadFileType, importOptionsPrep, UploadType.DEFAULT, keyToSend, (res) => {
+            const credentialsAndUploadId = JSON.parse(JSON.parse(res));
```

Comment:
double json parse usually indicates something isn't used correctly => maybe worth changing in backend, can be skipped if to complicated